tests/test_core.py

```diff
@@ -2863,6 +2863,11 @@ def test_time_c(self):
   def test_gmtime(self):
     self.do_core_test('test_gmtime.c')
 
+  def test_gmtime_localtime(self):
+    tz = 'EST+05'
+    with env_modify({'TZ': tz}):
+      self.do_core_test('test_gmtime_localtime.c')
```

Comment:
We already have `test_time.c` above that contains many tests for this function.  Why create new/separate test here?

Reply:
``test_time.c`` is executed multiple times with different settings for TZ.